modules/mco-update-boot-images.adoc

```diff
@@ -0,0 +1,159 @@
+
+:FeatureName: Boot image updating
+include::snippets/technology-preview.adoc[]
+
+You can view the current boot image used in your cluster by examining a machine set:
+
+.Example machine set with the boot image reference
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  name: ci-ln-hmy310k-72292-5f87z-worker-a
+  namespace: openshift-machine-api
+spec:
+# ...
+  template:
+# ...
+    spec:
+# ...
+      providerSpec:
+# ...
+        value:
+          disks:
+          - autoDelete: true
+            boot: true
+            projects/rhcos-cloud/global/images/rhcos-411-85-202203181601-0-gcp-x86-64 <1>
+# ...
+----
+<1> This boot image is the same as the installed {product-title} version, in this example, {product-title} 4.11. 
+
+If you configure your cluster to update your boot images, the boot image referenced in your machine sets is the same version as the cluster.
+
+If want to you opt out of this feature after some nodes have been updated with the new boot image, the nodes retain the updated boot image. There is no way to roll back the nodes to the original boot image version. Any new nodes created after opting out are created with the original boot image. Note that the boot image referenced in the machine set is not reverted, and continues to show the current cluster version.
+
+.Prerequisites
+
+* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates. For more information, see _Enabling features using feature gates_.
+
+.Procedure
+
+. Enable updating boot images by editing the `MachineConfiguration` object:
++
+[source,terminal] 
+----
+$ oc edit MachineConfiguration
+----
+
+.. Optional: Configure the boot image update for all machine sets:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: MachineConfiguration
+metadata:
+  name: cluster
+  namespace: openshift-machine-config-operator
+spec:
+# ...
+  managedBootImages: <1>
+    machineManagers: 
+    - resource: machinesets
+      apiGroup: machine.openshift.io
+      selection:
+        mode: All <2>
+----
+<1> Opts into boot image updating.
+<2> Specifies that all machine sets in the cluster will be updated.
+
+.. Optional: Configure the boot image updating for specific machine sets
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: MachineConfiguration
+metadata:
+  name: cluster
+  namespace: openshift-machine-config-operator
+spec:
+# ...
+  managedBootImages: <1>
+    machineManagers:
+    - resource: machinesets
+      apiGroup: machine.openshift.io
+      selection:
+        mode: Partial
+          partial:
+            machineResourceSelector:
+              matchLabels:
+                update-boot-image: "true" <2>
```

Comment:
looks like the preview takes out the quotes around true, oof

Reply:
It looks OK now.